app/models/recurring_todos/abstract_repeat_pattern.rb

```diff
@@ -174,7 +174,7 @@ def continues_recurring?(previous)
     # same day as the previous
     def determine_start(previous, offset=0.day)
       start = self.start_from || NullTime.new
-      now = Time.zone.now
+      now = Time.now
```

Comment:
I see, so trying to leave the ```Time.zone.now``` in I experience a weird thing. When I launch the todo_test alone it works, when I launch the full suite it breaks.
If you try printing this in the ```test_load``` method:
```
p @not_completed1.created_at
p 1.week.ago.beginning_of_day
```
you will get two different results :confused:

It looks like the TZ is not loaded correctly when I launch a single test?
Is maybe a problem of libfaketime? Do you have any idea? Is maybe a clue to fix the problem?